source/sgp_mode/Tasks.h

```diff
@@ -30,6 +32,11 @@ struct Task {
   std::variant<InputTask, OutputTask> kind;
   bool unlimited = true;
   emp::vector<size_t> dependencies;
+  uint32_t curHostOutput = 0;
+  uint32_t curSymOutput = 0;
+  std::map<uint32_t, uint32_t> hostCalculationTable;
+  std::map<uint32_t, uint32_t> symCalculationTable;
+  emp::vector<uint32_t> fullSquareList;
```

Comment:
Putting these here makes it so that all the tasks have them, which doesn't seem to make a lot of sense. You should check with Sylvie on how to set it up so that only the square task has these components